java/ql/src/Security/CWE/CWE-020/ExternalAPIsUsedWithUntrustedData.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Frequency counts for external APIs that are used with untrusted data
+ * @description This reports the external APIs that are used with untrusted data, along with how
+ *              frequently the API is called, and how many unique sources of untrusted data flow
+ *              to it.
+ * @id java/count-untrusted-data-external-api
+ * @kind table
```

Comment:
Would it make sense to tag this one with CWE-20 too so that the CWE reference was automatically added?

Reply:
Sure, let me see if I can do that as a suggestion:
```suggestion
 * @kind table
 * @tags security external/cwe/cwe-20
```